core/src/main/java/org/apache/brooklyn/core/catalog/internal/BrooklynToscaTags.java

```diff
@@ -0,0 +1,57 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.apache.brooklyn.core.catalog.internal;
+
+import com.google.common.base.Objects;
+
+/**
+ * Strongly types tag used to store TOSCA metadata such as:
+ * <ul>
+ *     <li>TOSCA type</li>
+ *     <li>Requirements</li>
+ *     <li>relationships</li>
+ * </ul>
+ */
+public class BrooklynToscaTags {
+
+    private String derivedFrom = "brooklyn.nodes.SoftwareProcess";
```

Comment:
This is the default `root` node defined in `brooklyn-tosca` which has a `host` requirement. See [here](https://github.com/cloudsoft/brooklyn-tosca/blob/master/a4c-brooklyn-plugin/src/main/resources/brooklyn/brooklyn-resources/brooklyn-resources.yaml)